.github/workflows/performance-benchmark.yml

```diff
@@ -0,0 +1,62 @@
+name: github-benchmarks
+on:
+  push:
+    branches:
+      - develop
+  pull_request:
+
+jobs:
+  CI:
+    continue-on-error: true
+    strategy:
+      matrix:
+        distro: ['ubuntu:latest']
+        cxx: ['g++', 'clang++']
+        backend: ['OPENMP']
+    runs-on: ubuntu-latest
+    container:
+      image: ghcr.io/kokkos/ci-containers/${{ matrix.distro }}
+    env:
+      BUILD_ID: ${{ matrix.distro }}-${{ matrix.cxx }}-${{ matrix.backend }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - uses: actions/cache@v3
+        with:
+          path: ~/.ccache
+          key: kokkos-${{ matrix.distro }}-${{ matrix.cxx }}-${{ matrix.openmp }}-${github.ref}-${{ github.sha }}
+          restore-keys: kokkos-${{ matrix.distro }}-${{ matrix.cxx }}-${{ matrix.openmp }}-${{github.ref}}
+      - name: Configure Kokkos
+        run: |
+          cmake -B builddir \
+            -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
```

Comment:
Please move it next to the other CMAKE_CXX_ option

Reply:
Done (https://github.com/kokkos/kokkos/commit/7e651ca70ba3c9df2c89da71e54a8fd5c9124309).